jekyll/_style/style-screenshots.adoc

```diff
@@ -0,0 +1,15 @@
+= Using screenshots
+:page-layout: classic-docs
+:icons: font
+:toc: macro
+:toc-title:
+
+* If you’re adding a screenshot, make it look nicer by sticking it in here: https://browserframe.com/. Choose **Generic Light** and drag and drop your screenshot into the window, then download and you will get a PNG.
```

Comment:
Could we say something like, "make the image easier to see / more scannable by adding a browser frame using browserframe.com" or something along those lines? It totally does make screenshots nicer and more consistent but I'd prefer providing a more functional reason 😁

Reply:
Yes good call!